rdrf/rdrf/views/registration_rdrf.py

```diff
@@ -97,6 +101,38 @@ def registration_allowed(self):
         return False
 
 
+@method_decorator(xframe_options_exempt, name='dispatch')
+class EmbeddedRegistrationCompletedView(TemplateView):
+    pass
+
+
+@method_decorator([xframe_options_exempt, csrf_exempt], name='dispatch')
```

Comment:
I can have another look, I think it can probably work without it (or maybe with something less severe) since trrf and the website are on the same domain (excuse any misplaced terminology there). Might be hard to test though before it's deployed.